packages/eslint-plugin/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+module.exports = {
+  extends: ['../../.eslintrc.js', 'plugin:eslint-plugin/recommended'],
+};
```

Comment:
For consistency with other packages, what about using `docusaurus-eslint-plugin`?

---

Does it mean it extends our root eslint config? Why do we want that? If we change a rule for ourselves, will this impact our users?

Will it work for users, considering they will only npm install individual packages and won't have our `<repoRoot>/.eslintrc.js` available locally?

Reply:
@slorber This is the configuration file for our own ESLint tooling. This means it will extend `eslint-plugin-eslint-plugin` which is recommended for all ESLint plugins.